apps/cowswap-frontend/project.json

```diff
@@ -71,6 +72,28 @@
       "options": {
         "buildTarget": "cowswap-frontend:build"
       }
+    },
+
+    "i18n": {
+      "executor": "nx:noop",
+      "dependsOn": ["i18n:compile"]
+    },
+    "i18n:compile": {
+      "executor": "nx:run-commands",
+      "dependsOn": ["i18n:extract"],
+      "options": {
+        "command": "npx lingui compile"
+      },
+      "inputs": ["{projectRoot}/src/locales/*.po"],
+      "outputs": ["{projectRoot}/src/locales/*.js"]
+    },
```

Comment:
Why not use a single command since `i18n` is a noop?

Reply:
good point. I guess I wanted to be explicit and replicate what we had. i18n will be calling for both extract and compile. I will delete one to reduce the number of items there

Reply:
1a810df0a9b0f13b22a024b28c72a709cd21ba65